//manejo de errores

//excepcion

/* 
let myObject
console.log(myObject.email) */

//--captura d errores
//try-catch

try{
    //codigo que va a probar
    console.log(myObject.email)
    console.log("finaliza la ejeccino sin errores")
} catch{
    //bloque de error
    console.log("finaliza la ejeccino con errores")
}